test/e2e/main_test.go

```diff
@@ -344,6 +344,32 @@ func TestManualMeteringInstall(t *testing.T) {
 			},
 			MeteringConfigManifestFilename: "s3.yaml",
 		},
+		{
+			Name:                      "ValidateNFS-ReportDynamicInputData",
+			MeteringOperatorImageRepo: meteringOperatorImageRepo,
+			MeteringOperatorImageTag:  meteringOperatorImageTag,
+			Skip:                      !runAllInstallTests,
```

Comment:
Unfortunately, this needs to be set to `Skip: false` or else the e2e suite for this PR won't run this test. We may want to revert back to `Skip: !runAllInstallTests` once we get closer to cleaning this PR up.